correct description of `skipMiddlewareUrlNormalize` in advanced middleware flags (#60841)
vercel/next.js: docs/02-app/01-building-your-application/01-routing/13-middleware.mdx

}
```

> `skipMiddlewareUrlNormalize` allows disabling URL normalizing in Next.js to make handling direct visits and client-transitions the same. There are some advanced cases where you need full control using the original URL which this unlocks.

Reviewer: ```suggestion
`skipMiddlewareUrlNormalize` allows for disabling the URL normalization in Next.js to make handling direct visits and client-transitions the same. In some advanced cases, this option provides full control by using the original URL.
```
